relay-server/src/actors/project_buffer.rs

```diff
@@ -341,8 +358,17 @@ impl BufferService {
 
             let buf = std::mem::take(buffer);
             self.count_mem_envelopes = 0;
-            Self::do_spool(db, buf).await?;
```

Comment:
On the line `351` we also get estimated database file size. I think it also would be great to have a metric on that, to see how the file grow with time. What do you think, does it make sense?

Reply:
On one side yes, but it'll also give us at least some overview what's going on with the system I guess.